README.md

```diff
@@ -0,0 +1,110 @@
+# Flynn Discovery
+
+A cluster peer discovery server.
+
+`flynn-discovery` provides a simple HTTP interface to create clusters, register cluster members and get cluster members.
+
+To install:
+
+```
+$ go get github.com/flynn/flynn-discovery
+$ cd $GOPATH/src/github.com/flynn/flynn-discovery
+$ go build
+```
+
+`flynn-discovery` requires a Postgres database to save cluster information. For this demo we will use a Docker container.
```

Comment:
How about deploy it to a Flynn cluster which has Postgres already running :wink: